python/ray/train/tensorflow.py

```diff
@@ -79,3 +81,19 @@ def handle_failure(self, worker_group: WorkerGroup,
             worker_group.execute(shutdown_session)
         worker_group.add_workers(len(failed_worker_indexes))
         self.on_start(worker_group, backend_config)
+
+
+def prepare_dataset_shard(tf_dataset_shard: tf.data.Dataset):
+    """ A utility function that disables Tensorflow autosharding
+    since the dataset has already been sharded.
```

Comment:
```suggestion
    """ A utility function that disables Tensorflow autosharding.
    
    This should be used on a  TensorFlow ``Dataset`` created by calling ``to_tf()`` 
    on a ``ray.data.Dataset`` returned by ``ray.train.get_dataset_shard()`` since 
    the dataset has already been sharded across the workers.
```

1. First sentence should be limited to one line.
2. Make it explicit that this is to be used for the Ray Datasets integration.